habitat-hitl/habitat_hitl/core/remote_client_state.py

```diff
@@ -246,6 +246,9 @@ def _update_input_state(
             if len(client_states) == 0:
                 continue
             gui_input = self._gui_inputs[user_index]
+            mouse_scroll_offset: float = 0.0
+            relative_mouse_position: List[int] = [0, 0]
```

Comment:
OK, so this is so continuous inputs do not accumulate but continue like a platform-shaped step-change.

Reply:
What's going on here is the following:
* When the server steps (simulator set), multiple client messages may have been received.
* If we just pick the last received value, we lose discrete information like clicks.
* While not as critical, it follows that the correct aggregation strategy for continuous input is accumulating them.